probabilistic.go

```diff
@@ -165,6 +169,79 @@ func (c cmdable) BFExists(ctx context.Context, key string, element interface{})
 	return cmd
 }
 
+func (c cmdable) BFLoadChunk(ctx context.Context, key string, iterator int64, data interface{}) *StatusCmd {
+	args := []interface{}{"bf.loadchunk", key, iterator, data}
```

Comment:
Given history (and nothing else) commands really should be all in CAPS everywhere. Yes, we're not doing that currently, but no time like the present.